packages/analytics-js/src/services/StoreManager/Store.ts

```diff
@@ -207,10 +209,10 @@ class Store implements IStore {
     }
 
     const extensionPointName = `storage.${mode}`;
+    const formattedValue = this.pluginManager ?
+      this.pluginManager.invokeMultiple<string>(extensionPointName, value) : value;
```

Comment:
Should this be `invokeSingle` as we use only the first result?

Reply:
yes need to go thorugh all extension points in our code and replace with single where needed, At the moment all i
are invokeMultiple to match the previos behavior.

Once we have all plugins working we can decide on what extension points we potentially would like to allow multible
and single ones

Reply:
done for this one